manager/assets/modext/widgets/fc/modx.panel.fcprofile.js

```diff
@@ -107,19 +103,19 @@ MODx.panel.FCProfile = function(config) {
 };
 Ext.extend(MODx.panel.FCProfile,MODx.FormPanel,{
     initialized: false
+
     ,setup: function() {
         if (!this.initialized) { this.getForm().setValues(this.config.record); }
         if (!Ext.isEmpty(this.config.record.name)) {
-            Ext.defer(function() {
-                Ext.getCmp('modx-fcp-header').update(_('profile') + ': ' + Ext.util.Format.htmlEncode(this.config.record.name));
-            }, 250, this);
+            Ext.getCmp('modx-fcp-header').html = Ext.util.Format.htmlEncode(this.config.record.name);
```

Comment:
This is incorrect, should be: `Ext.getCmp("modx-header-breadcrumbs").updateHeader(Ext.util.Format.htmlEncode(this.config.record.name));`

Reply:
I did, as here - https://github.com/modxcms/revolution/pull/14890/files?file-filters%5B%5D=.js#diff-4fa92d4eddfe372ac291e0e50ad38ea4R161
The version with `.updateHeader` in some cases throws an error, but with `.html` this was not observed.
And what is the need to change the class to `modx-header-breadcrumbs`?

Reply:
You're not changing class, you're grabbing different component - the one that has the updateHeader method. What error and in which case you get when using updateHeader? I just tried replacing it in the file you sent and it works without an error...

Reply:
Apparently I confused and did not change the classes to `.modx-header-breadcrumbs` due to which `updateHeader()` caused an error.

Reply:
Again, it's not a class, it's an id of component you are retrieving. And the updateHeader function is available only on `modx-header-breadcrumbs` component.

Reply:
Ok, I will correct my and your files.

Reply:
thanks 👍